Src/main.c

```diff
@@ -72,6 +72,12 @@ int main(void)
 
   /* USER CODE BEGIN 1 */
 
+    // Variables. 
+    unsigned short potentiometerValue;
+    unsigned short digitalValue;
```

Comment:
initialize to 0 please. We usually never leave uninitialized variables lying around cause they can cause issues (fun fact, I once spent 5 hours debugging something for WARG to find out I forgot to initialize a variable)